bin/tmuxinator

```diff
@@ -4,16 +4,10 @@ $: << File.expand_path("../../lib/", __FILE__)
 require "thor"
 require "tmuxinator"
 
-if ARGV.length == 1
-  name = ARGV[0]
+name = ARGV[0]
 
-  if Tmuxinator::Cli::COMMANDS.keys.include?(name.to_sym)
-    Tmuxinator::Cli.start
-  elsif Tmuxinator::Config.exists?(name)
-    Tmuxinator::Cli.new.start(name)
-  else
-    Tmuxinator::Cli.start
-  end
+if !Tmuxinator::Cli::COMMANDS.keys.include?(name.to_sym) && Tmuxinator::Config.exists?(name)
```

Comment:
Please add a guard against a `nil` name here. Running `mux` should just show the help.